sonic-pcied/scripts/pcied

```diff
@@ -53,6 +54,49 @@ class DaemonPcied(DaemonBase):
         self.state_db = swsssdk.SonicV2Connector(host=REDIS_HOSTIP)
         self.state_db.connect("STATE_DB")
 
+    # Load AER-fields into STATEDB
+    def update_aer_to_statedb(self, device, device_name, pcie_hash_name):
```

Comment:
Can we have a separate platform api to get the aer status on a specific device? And have a seperate api to update the state_db here in daemon?

Reply:
Based on Azure/SONiC#702, new commit has been pushed to use `get_pcie_aer_stats` added by Azure/sonic-platform-common#144